### ISSUE TITLE
Clarification: 6.6.3.2 Receiving a cancelOrder action when AGV has no order

### ISSUE THREAD

**Issue by Mehno** [VDA5050/VDA5050#345]
How is "has no Order" defined? Does it mean that the vehicle has neither node nor edge states and all action states are in an end state?
Or does it mean that the vehicle has never received an order? 

Is there an explicit definition in the standard? If not one should be provided.

**Comment by collani-bosch** [comment]
Correct: no node and edge states and existing action states of an order (!) are either FINSHED or FAILED. This indicates that the vehicle isn't executing an order. (It's not equal to "the order is finished", because this knowledge has only MC).
This is described in the context of figure 8.

**Comment by Mehno** [comment]
But the wording "has no order"  is not defined in figure 8. Just "executing and order". Maybe it is just the mathematician in me. :D

**Comment by ConstantinUp** [comment]
If we need a clarification, I would propose a simple clarification, according to the answer of @collani-bosch:

`A vehicle has no order, if the node and edge states are empty and all actions that were part of an order (not the received instantActions) are either 'FINISHED' or 'FAILED'. This order can be continued or a new order can be sent as shown in Figure 8.`

**Comment by ConstantinUp** [comment]
In version 3.0 we clarified that using the idle state (see chapter 6.6.8 (https://github.com/VDA5050/VDA5050/blob/dev/V3.0.0/VDA5050_EN.md#668-idle-state-of-the-mobile-robot). Please reopen if questions remain.
